app/models/application.rb

```diff
@@ -0,0 +1,24 @@
+class Application < ApplicationRecord
+  has_many :application_pets
+  has_many :pets, through: :application_pets
+  has_many :shelters, through: :pets
+  validates :name, presence: true
+  validates :street_address, presence: true
+  validates :city, presence: true
+  validates :state, presence: true
+  validates :zip_code, presence: true
+  validates :endorsement, presence: true
+
+  def find_pets
+    @pets = Pet.joins(:applications).where("applications.id = #{self.id}")
+  end
```

Comment:
Since this method is not passing anything to a view, there is no need for an instance variable.

Beyond that, if the intent of this method is to get all the pets on this application, you already have that info through your association on line 3.